fuzz/src/full_stack.rs

```diff
@@ -134,10 +134,18 @@ impl<'a> std::hash::Hash for Peer<'a> {
 	}
 }
 
+type ChannelMan = ChannelManager<
+	EnforcingChannelKeys,
+	Arc<channelmonitor::SimpleManyChannelMonitor<OutPoint, EnforcingChannelKeys, Arc<TestBroadcaster>, Arc<FuzzEstimator>, Arc<dyn Logger>, Arc<ChainWatchInterfaceUtil>>>,
+	Arc<TestBroadcaster>, Arc<KeyProvider>, Arc<FuzzEstimator>, Arc<dyn Logger>>;
+type PeerMan = PeerManager<Peer<'a>, Arc<ChanMan>,
```

Comment:
I think this generic type declaration doesn't compile. Also see `CannelMan` or other compilation error nits.